lib/rpush/client/redis/notification.rb

```diff
@@ -57,6 +57,7 @@ def self.absolute_retryable_namespace
         attribute :retailer_customer_id, :string
         attribute :queue_to_front, :boolean, default: false
         attribute :seed_list_send, :boolean, default: false
+        attribute :cluster, :string
```

Comment:
I'm thinking if we should also add this to `Rpush::Client::Redis::App` model and in `Rpush::Daemon::Synchronizer.sync`, only start dispatchers for the relevant apps. Probably won't make a difference with only a few clusters, but may be an optimization as we have more

Reply:
Makes sense